.github/workflows/supabase_flutter_ci.yml

```diff
@@ -32,7 +32,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        flutter-version: ['3.0.0', '3.x']
+        flutter-version: ['3.10.x', '3.13.x']
```

Comment:
Should we keep the `3.x` to test it against the latest Flutter version?

Reply:
Hm yeah. I've replaced 3.13.x now with 3.x, because they are currently the same.